src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,32 @@
+import { Person } from '../../types/Person';
+import './Autocomplete.scss';
+
+interface Props {
+  delay: number;
+  filteredPeople: Person[];
+  onSelected: (person: Person | null) => void;
+}
+
+export const Autocomplete: React.FC<Props> = ({
+  // delay,
+  filteredPeople,
+  onSelected,
+}) => {
+  return (
+    <>
+      {filteredPeople.map((person: Person) => (
+        <div
+          className="dropdown-item"
+          data-cy="suggestion-item"
+          key={person.slug}
```

Comment:
Ensure that `person.slug` is a unique and stable identifier for the `key` prop. This prevents issues with React's reconciliation process.